src/main/java/homework09/ExtraMainFutureTask.java

```diff
@@ -0,0 +1,49 @@
+package homework09;
+
+import java.util.concurrent.Callable;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.FutureTask;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+
+public class ExtraMainFutureTask {
+
+	public static void main(String[] args) {
+		ExecutorService es = Executors.newFixedThreadPool(2);
+		
+		Callable<String> callable1 = new CountThread(300);
+		Callable<String> callable2 = new CountThread(1000);
+
+		FutureTask<String> ft1 = new FutureTask<String>(callable1);
+		FutureTask<String> ft2 = new FutureTask<String>(callable2);
+
+		es.execute(ft1);
```

Comment:
ExecutorService uses method submit() for a Callable implementations.

Reply:
> ExecutorService uses method submit() for a Callable implementations.

Yea, I know. I use submit with Future in the next file containing Main method, but this is another example not really mentioned in home task. It is FutureTask, I just did this Main for myself to learn : ) Maybe I shouldn't add this file to commit, my bad.